src/typings/globalize.d.ts

```diff
@@ -34,6 +34,4 @@ interface GlobalizeStatic {
     findClosestCulture(cultureSelector: string): GlobalizeCulture;
 }
 
-/* tslint:disable:no-var-keyword */
-var Globalize: GlobalizeStatic = Globalize || window["Globalize"];
-/* tslint:enable */
+declare var Globalize: GlobalizeStatic;
```

Comment:
Did you test this library against Power BI? I suppose that the ```Globalize``` isn't available in the global scope due to faked window object.